src/routes/api/user-login.routes.js

```diff
@@ -0,0 +1,11 @@
+
+import { Router } from 'express';
+import { requireAuth } from '../../middlewares/requireAuth.js';
+import { loginUser, protectedRouteHandler } from '../../controllers/user-login-controller.js';
+
+const LoginRouter = Router();
+
+LoginRouter.post('/users/login', loginUser);
```

Comment:
you're not following the our folder structure convention, we suppose to only have single routes for users.
why did you added new file?